From 5a3cb9de549ba4a012c93b4288e9f3e48f492053 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 9 Aug 2016 10:43:03 -0700
Subject: [PATCH] RELENG-705 purge-mg-builds in MG needs node-manta with 'mls
 --type d ...' support Reviewed by: Matt Smillie <matt.smillie@joyent.com>
 Reviewed by: Julien Gilli <julien.gilli@joyent.com>

---
 package.json          | 2 +-
 tools/purge-mg-builds | 4 ++++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/package.json b/package.json
index e5e207e..f58dc08 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "manta": "1.4.3",
+    "manta": "3.0.0",
     "smartdc": "7.3.0",
     "imgapi-cli": "git+ssh://git@github.com:joyent/sdc-imgapi-cli.git#38332b1"
   },
diff --git a/tools/purge-mg-builds b/tools/purge-mg-builds
index 589adda..964ec17 100755
--- a/tools/purge-mg-builds
+++ b/tools/purge-mg-builds
@@ -65,6 +65,10 @@ set -o pipefail
 
 #---- globals, config
 
+# Get `mls` et al on the PATH.
+TOP=$(cd $(dirname $0)/../ >/dev/null; pwd)
+export PATH=$TOP/node_modules/.bin:$PATH
+
 #
 # A mapping from build name to the number of days' builds to keep.
 #
-- 
2.21.0

